packages/design-system/.storybook/components.tsx

```diff
@@ -1,6 +1,20 @@
-import type { GlobalProvider } from '@ladle/react';
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// biome-ignore lint/correctness/noUnusedImports: fix later
+import React from 'react';
```

Comment:
Why is this import necessary?

Reply:
![Screenshot 2025-01-15 at 8 30 44 AM](https://github.com/user-attachments/assets/5a68bea7-4575-4b84-bdbe-995201439f6a)

ts complains without it - seems like a config issue. was planning on looking into it this am

Reply:
updated here: https://github.com/gohypergiant/standard-toolkit/pull/167/commits/47318c45b5c96c3f8017dcd3f5a87ad99499423c